packages/jupyter-ai/jupyter_ai/chat_handlers/fix.py

```diff
@@ -64,22 +65,23 @@ class FixChatHandler(BaseChatHandler):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.prompt_template = None
 
     def create_llm_chain(
         self, provider: Type[BaseProvider], provider_params: Dict[str, str]
     ):
         unified_parameters = {
+            "verbose": True,
```

Comment:
When I call `/fix` on the error cell below:

<img width="431" alt="Screenshot 2024-10-25 at 2 37 13 PM" src="https://github.com/user-attachments/assets/fb8e883e-b2e8-4e0f-b21b-330cc48f6be7">

`jupyter-lab` logs the following into stdout:

````
usage: jupyter-lab [-h]
jupyter-lab: error: unrecognized arguments:

```
x = 1
assert x = 1
```
````

Since specifying `"verbose": True` doesn't log the full prompt for debugging and also causes a weird error to be logged, can this change be reverted?

Reply:
This is not an issue with `fix.py` and is occurring in `base.py` when handling arguments, will need to be addressed separately from this PR.